code/WorkInProgress/recycling/conveyor.dm

```diff
@@ -107,6 +107,8 @@
 		return
 	if(istype(A, /obj/critter) && A:flying)		//They are flying above it, ok.
 		return
+	if(isflock(A))
+		return
```

Comment:
(minor) Feels like this should be an atom prop

Reply:
There is an atom prop, `ATOM_PROP_FLOCK_THING`. It's used to marked converted things (tiles, computers, etc), while isflock (as you reviewed earlier) is for mobs.

Reply:
meant the conveyor immunity, was intentionally including 108-109 (though that isn't conveyed well since it's 4 lines minimum anyways. grumble)
alternately `PROP_MOB_FLOATING` or whatever in case that comes up in other places

Reply:
living object code strikes again, of course everything is a mess of spaghetti when I go to try and do this. There's something similar with machinery bots, which are ignored when they're 'on'. great.